WooCommerce/Classes/ViewRelated/Orders/FulfillViewController.swift

```diff
@@ -166,6 +166,8 @@ extension FulfillViewController {
 
             WooAnalytics.shared.track(.orderStatusChangeFailed, withError: error)
             DDLogError("⛔️ Order Update Failure: [\(orderID).status = \(status.rawValue)]. Error: \(error)")
+
+            self.displayErrorNotice(orderID: orderID)
```

Comment:
I think we should have a weak reference to `self` on the `return statement ☝️ just to be safe.

Reply:
I considered that myself as well. But I noticed that if I weakifyed the reference to self, by the time this closure is executed, that reference would be nil.

I think that might be because the `Action` returned by the `updateOrderAction` is a local variable in the `fulfillWasPressed` method, so it does not get retained by `FulfillViewController`

I did run this as it is through instruments and it did not seem to leak. 

I do agree though that it would be safer to weakify self in that closure. I will play with it a bit more and see if I can find a better solution. Thanks!